gammapy/time/plot.py

```diff
@@ -30,3 +32,167 @@ def plot_time_difference_distribution(time, ax=None):
 
     # TODO: implement!
     raise NotImplementedError
+
+
+def plot_fermi_3fgl_light_curve(name_3fgl, time_start=None, time_end=None, ax=None):
+    """Plot flux as a function of time for a fermi 3FGL object.
+
+    Parameters
+    ----------
+    name_3FGL : `string`
+        The 3FGL catalog name of the object to plot
+    time_start : `~astropy.time.Time`
+        Light curve start time.  If none, use the earliest time in the catalog.
+    time_end : `~astropy.time.Time`
+        Light curve end time.  If none, use the latest time in the catalog.
+    ax : `~matplotlib.axes.Axes` or None
+        Axes
+
+    Returns
+    -------
+    ax : `~matplotlib.axes.Axes`
+        Axes
+
+    Examples
+    --------
+    Plot effective area vs. energy:
+
+    .. plot::
+        :include-source:
+
+        from astropy.time import Time
+        from gammapy.time import plot_fermi_3fgl_light_curve
+        import matplotlib.pyplot as plt
+
+        time_start = Time('2010-01-01T00:00:00')
+        time_end = Time('2015-02-02T02:02:02')
+
+        plt.plot = plot_fermi_3fgl_light_curve('3FGL J0349.9-2102', time_start, time_end)
```

Comment:
You shouldn't assign to `plt.plot` ... that's a function in matplotlib that you're overwriting with the axes object returned here.
Here there's no need to store the axes object in a variable, because you don't want to modify the plot further, so just remove `plt.plot = `.